Some systems define errno as a macro, and (I think that) C99 forbids to have
an identifier called 'errno'. Rename vh->errno to error.
commit eb7014c3 upstream.

diff --git src/config.c src/config.c
index 2015129..4a38851 100644
--- src/config.c
+++ src/config.c
@@ -54,7 +54,7 @@ struct viam_handle {
   viam_bank_t banks;		/**< list of banks */
   viam_filter_t filters;	/**< list of filters */
 
-  int errno;			/**< last error code */
+  int error;			/**< last error code */
 };
 
 
@@ -163,11 +163,11 @@ viam_config_setfilters(viam_handle_t vh, viam_filter_t filter)
 viam_errorcode_t
 viam_errno(viam_handle_t vh)
 {
-  return vh->errno;
+  return vh->error;
 }
 
 int
 viam_seterrno(viam_handle_t vh, int eno)
 {
-  return vh->errno = eno;
+  return vh->error = eno;
 }
